internal/http/request.go

```diff
@@ -36,34 +34,24 @@ func NewRequest(c Client, method string, url string, params interface{}, reqBody
 		authStrategy: c.authStrategy,
 	}
 
+	// FIXME: We should remove this requirement on the request
```

Comment:
We should decide as a team on TODO or FIXME.. I prefer TODO as some IDEs/editor plugins have built-in support for TODO

Reply:
Either way. `vim` highlights both 😄